### DISCUSSION TITLE
Refund

### DISCUSSION THREAD

**Discussion by apzpn** [community/community#40709] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Hi 
I forget to cancel my free trial of copilot.
Then I payed 10$. I canceled my subscription and I do not want use copilot anymore. How can I ask refund please.

**Comment by davecheney** [comment]
Please contact support who can assist you https://support.github.com/contact

**Comment by Akash1134** [comment, marked as answer]
We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.

**Reply by arajay** [reply]
i think this may have changed; for me the "virtual assistant" closes as soon as i visit the page and the only option is to submit a support ticket

**Reply by sth4me** [reply]
Me too, I cant see any 'Virtual Agent'。

**Comment by UndaRatedApp** [comment]
Hi I have been charged recently and I don't require the services can I get a refund please

**Reply by InWamos** [reply]
Why not reaching the support with this question?

**Comment by mihailthebuilder** [comment]
Just wanted to mention that the provided answer is correct. I tried it today, and was given a refund of $7.70 in about a minute. Not the entire $10, I'm guessing because a week had gone by since the payment was made, even though I didn't use it.
